doc/design/assumeutxo.md

```diff
@@ -63,7 +63,7 @@ chainstate and a sync to tip begins. A new chainstate directory is created in th
 datadir for the snapshot chainstate called `chainstate_snapshot`.
 
 When this directory is present in the datadir, the snapshot chainstate will be detected
-and loaded as active on node startup (via `DetectSnapshotChainstate()`).
+and loaded as active on node startup (via `DetectAssumeutxoChainstate()`).
```

Comment:
in 21d638c92f56f5ad4d32c47240700b41e8868789

nit: `DetectUnvalidatedChainstate` would have been more consistent with the rest of the naming choices in this commit